protocols/rendezvous/src/codec.rs

```diff
@@ -586,4 +677,55 @@ mod tests {
 
         assert_eq!(bytes.len(), 8 + 3)
     }
+
+    #[async_std::test]
+    async fn codec_test() {
```

Comment:
I don't think we need this additional test, we are just composing additional functionality for the codecs.